test/legacy/reconfigure.js

```diff
@@ -62,6 +62,15 @@ describe('Reconfigure', function() {
     done();
   });
 
+  // Issue cloudant/nodejs-cloudant#323
+  it('allows a username, a password, and url to be passed in', function(done) {
+    var credentials = { username: 'myusername', password: 'mypassword', url: 'http://localhost:8081' };
+    var outCreds = reconfigure(credentials);
+    outCreds.outUrl.should.be.a.String;
+    outCreds.outUrl.should.equal('http://myusername:mypassword@localhost:8081');
+    done();
+  });
+
```

Comment:
Perhaps another test where the `url` passed contains credentials as well?
This will validate the override behaviour, which is that `username` and `password` options supersede credentials supplied in a `url`.
I don't know if there is a reasonable place to document that as well - maybe in the `README` `Initialization>Using a URL` section?